src/Exception/NotFoundException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+
+namespace Tuf\Exception;
+
+use Throwable;
+
+class NotFoundException extends \Exception
```

Comment:
Looks like I didn't end up using this in the final version of this PR. Happy to remove it, but OTOH we're gonna need things like this pretty soon.